### PR TITLE
add theme support

### PR DESCRIPTION
**Pull request by mehdiraized**
## TODO (after merge)

- plz update library version

### CODE REVIEW

**Comment by mehdiraized** [comment]
@mk-emeks plz check

File: README.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ Q1: Which variables can I override?
 
 > The variables that are exposed by antd in this file: https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less
 
+`You Can Change with theme Parametr`
```
Comment (randomeduc) [inline README.md]: There's a little typo in Parametr and I think that is not necessary to capitalize each word.

It would be nice to reference to the antd theme folder so the user know where to find the respective .less file (https://github.com/ant-design/ant-design/tree/master/components/style/themes). Maybe rephrase the previous paragraph, what do you think?

Comment (mehdiraized) [reply README.md:47]: My English is not good, please change it yourself

Comment (mk-emeks) [inline README.md]:
```suggestion
> Also, you could use `[theme]` parameter in order to target to a different theme such as `dark` or `compact` (To check all available options, please inspect this folder: https://github.com/ant-design/ant-design/tree/master/components/style/themes).
```

File: README.md [hunk 2]
```diff
@@ -62,6 +64,7 @@ Q3: Is there a way of changing command defaults?
 > - customThemeFilePath = "./custom-theme.less"
 > - generatedThemeFilePath = "./custom-theme.css"
 > - antdLibraryPath = "./node_modules/antd"
+> - theme = "defualt" You can use dark or compact
```
Comment (mk-emeks) [inline README.md]:
```suggestion
> - theme = "default"
```

**Review by mk-emeks — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mk-emeks) [reply README.md]: Also notice that few lines above

```
 $ generate-theme [verbose] [customThemeFilePath] [generatedThemeFilePath] [antdLibraryPath]
```

should be now:
```
 $ generate-theme [verbose] [customThemeFilePath] [generatedThemeFilePath] [antdLibraryPath] [theme]
```

File: README.md (hunk 1 above)
Comment (mk-emeks) [reply README.md]: @mehdiraized I suggested the expected changes so you could apply them in your branch =)

**Comment by randomeduc** [comment]
@mehdiraized `v0.1.5` with theme support is out. Thanks for your contribution! :smiley:
